downstream/titles/eda/eda-user-guide/docinfo.xml

```diff
@@ -1,7 +1,7 @@
-<title>Red Hat Event-Driven Ansible Controller user guide </title>
+<title>Event-Driven Ansible Controller user guide </title>
 <productname>Red Hat Ansible Automation Platform</productname>
 <productnumber>2.4</productnumber>
-<subtitle>Instructions on using Red Hat Event-Driven Ansible Controller</subtitle>
+<subtitle>Instructions on using Event-Driven Ansible Controller</subtitle>
 <abstract>
```

Comment:
There is now a module for providing feedback that Aine created. Change this text to "This guide provides instructions to help you set up and begin using Event-Driven Ansible Controller." or something similar. Refer to the abstract evaluation to see how it was handled for other docs. Also, I believe controller should be lowercase but I'm not positive. We've been instructed in the past to lower case "automation controller" but I'm not positive whether it applies here? 

Also suggest changing the subtitle to something active like: "Learn to configure and use Even-Driven Ansible Controller." or something along those lines.